OrbitCore/CrashHandler.cpp

```diff
@@ -4,13 +4,14 @@
 
 #include "CrashHandler.h"
 
+#include "client/settings.h"
+
 #include "Core.h"
 #include "OrbitBase/Logging.h"
 #include "OrbitDbgHelp.h"
 #include "ScopeTimer.h"
 #include "TcpClient.h"
 #include "Version.h"
-#include "client/settings.h"
```

Comment:
I this a clang-format related change?

Reply:
If not I would consider removing it.

Reply:
I changed it just to have a separate group with libraries' headers that are not from our project (`crashpad` library in this case).